packages/protobuf-test/src/gen/js/extra/deprecation-implicit_pb.d.ts

```diff
@@ -26,8 +27,47 @@ export declare const fileDesc_extra_deprecation_implicit: GenDescFile;
  * @deprecated
  */
 export declare type ImplicitlyDeprecatedMessage = Message<"spec.ImplicitlyDeprecatedMessage"> & {
+  /**
+   * @generated from field: string implicitly_deprecated_field = 1;
+   */
+  implicitlyDeprecatedField: string;
 };
 
 // Describes the message spec.ImplicitlyDeprecatedMessage. Use `create(ImplicitlyDeprecatedMessageDesc)` to create a new ImplicitlyDeprecatedMessage.
 export declare const ImplicitlyDeprecatedMessageDesc: GenDescMessage<ImplicitlyDeprecatedMessage>;
 
+/**
+ * @generated from enum spec.ImplicitlyDeprecatedEnum
+ * @deprecated
+ */
+export enum ImplicitlyDeprecatedEnum {
+  /**
+   * @generated from enum value: IMPLICITLY_DEPRECATED_ENUM_UNSPECIFIED = 0;
+   */
+  UNSPECIFIED = 0,
+}
+
+// Describes the enum spec.ImplicitlyDeprecatedEnum.
+export declare const ImplicitlyDeprecatedEnumDesc: GenDescEnum<ImplicitlyDeprecatedEnum>;
+
+/**
+ * @generated from service spec.ImplicitlyDeprecatedService
+ * @deprecated
+ */
+export declare const ImplicitlyDeprecatedService: GenDescService<{
+  /**
+   * @generated from rpc spec.ImplicitlyDeprecatedService.ImplicitlyDeprecatedRpc
+   */
+  implicitlyDeprecatedRpc: {
+    kind: "unary";
+    I: Empty;
+    O: Empty;
+  },
+}
+>;
+
+/**
+ * @generated from extension: int32 implicitly_deprecated_option = 2001;
+ */
+export declare const implicitly_deprecated_option: GenDescExtension<FieldOptions, number>;
```

Comment:
We're missing a `@deprecated` tag here, and also for message, enum, and file descriptors.

Improving this will be a follow-up.